src/components/AppVersionInfo.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+  <div class="section-header">
+    <div>
+      <h1>{{ $t('App') }}</h1>
+      <p class="overline">{{ "Version: " + appVersion }}</p>
+    </div>
+    <p class="overline">{{ "Built: " + getDateTime(appInfo.builtTime) }}</p>
+  </div>
+</template>
+  
+<script setup lang="ts">
+import { DateTime } from 'luxon';
+
+declare var process: any;
+
+const appInfo = (process.env.VUE_APP_VERSION_INFO ? JSON.parse(process.env.VUE_APP_VERSION_INFO) : {}) as any;
```

Comment:
We should move the logic to use the `app-version-info` package in dxp itself.

Reply:
We cannot use the App Version Info package in dxp as per my findings if we use app version info in dxp, we cannot access the env of appInfo created during the app version info Package run in the component. Also, I think we need App Version Info to execute commands during the app build. But as with this the git command will execute during dxp build and the info will be of no use.